src/CarbonAware.WebApi/test/integrationTests/mocks/WattTimeDataSourceMocker.cs

```diff
@@ -138,6 +174,17 @@ private void SetupResponseGivenGetRequest(string path, string body)
         );
     }
 
+    private void SetupResponseGivenPostRequest(string path, string body)
```

Comment:
I thought that all of the WattTime API endpoints were `GET` requests.   When would we plan on using this method?